test/integration/parachain/staging/sequential/replace.test.ts

```diff
@@ -80,70 +81,91 @@ describe("replace", () => {
         });
 
         it("should request vault replacement", async () => {
-            // try to set value above dust + estimated fees
-            const issueAmount = dustValue.add(feesEstimate).mul(1.2);
-            const replaceAmount = dustValue;
-            await issueSingle(
-                interBtcAPI,
-                bitcoinCoreClient,
-                userAccount,
-                issueAmount,
-                vault_3_id
-            );
+            for (const vault_3_id of vault_3_ids) {
+                // try to set value above dust + estimated fees
+                const issueAmount = dustValue.add(feesEstimate).mul(1.2);
+                const replaceAmount = dustValue;
+                await issueSingle(
+                    interBtcAPI,
+                    bitcoinCoreClient,
+                    userAccount,
+                    issueAmount,
+                    vault_3_id
+                );
+    
+                interBtcAPI.setAccount(vault_3);
+                await interBtcAPI.replace.request(
+                    replaceAmount, 
+                    currencyIdToMonetaryCurrency(vault_3_id.currencies.collateral) as CollateralCurrency
+                );
```

Comment:
Would be cleaner to refactor this to use the `callWith` utility.

Reply:
Ah, nice! Thank you for that. Added these to the follow up task, too.